README.md

```diff
@@ -52,3 +52,8 @@ To run python and JS tests with docker:
 
 * ```docker build -t shipit-test-runner -f Dockerfile-tests ./```
 * ```docker run --rm -v $(pwd):/src -ti shipit-test-runner /src/scripts/run_tests.sh```
+
+## Deployment
+After a PR is merged to master, it is automatically deployed to https://ship-it-dev.allizom.org/ and https://ship-it.allizom.org/. In order to deploy to production, the `master` branch should be pushed to the `production` branch:
```

Comment:
Nice to have it on https://ship-it.allizom.org/ too! It's so cool to have things automated!